pkg/v1/remote/write.go

```diff
@@ -41,17 +41,91 @@ type manifest interface {
 
 // Write pushes the provided img to the specified image reference.
 func Write(ref name.Reference, img v1.Image, options ...Option) error {
-	ls, err := img.Layers()
+	return WriteMulti([]name.Reference{ref}, img, options...)
+}
+
+// WriteMulti pushes the provided img to multiple specified image references.
+func WriteMulti(refs []name.Reference, img v1.Image, options ...Option) error {
```

Comment:
Writing multiple tags seemed to be expensive (digesting, compressing, whatever). Adding the ability to just push refs cut about 20 seconds at the end.

Reply:
I think we can partially solve some of this by just implementing an explicit tagging operation, see: https://github.com/google/go-containerregistry/issues/349

Reply:
Yep!